discovery-provider/src/tasks/vacuum_db.py

```diff
@@ -0,0 +1,34 @@
+import logging
+from src.tasks.celery_app import celery
+from psycopg2.extensions import ISOLATION_LEVEL_AUTOCOMMIT
+
+logger = logging.getLogger(__name__)
+
+
+@celery.task(name="vacuum_db", bind=True)
+def vacuum_db(self):
+    """Vacuum the db"""
+
+    db = vacuum_db.db
+    redis = vacuum_db.redis
+
+    have_lock = False
+    update_lock = redis.lock("vacuum_db", timeout=3600)
+
+    try:
+        have_lock = update_lock.acquire(blocking=False)
+
+        if have_lock:
+            connection = db.raw_connection()
```

Comment:
sorry i should have given you this context earlier. we do this one way here https://github.com/AudiusProject/audius-protocol/blob/master/discovery-provider/src/tasks/index_materialized_views.py#L7-L17. can we keep it consistent?